docs/topics/forms/modelforms.txt

```diff
@@ -526,7 +526,7 @@ For example, if you want the ``CharField`` for the ``name`` attribute of
 ``<input type="text">``, you can override the field's widget::
 
     from django.forms import ModelForm, Textarea
-    from myapp.models import Author
+    from .models import Author
```

Comment:
It looks like this feel hasn't had a new line added before the relative imports.
We should either not add new lines or add them everywhere - I don't have a strong opinion here

Reply:
I added a newline here to be consistent

Reply:
I spotted one more :+1: